charts/frontend/values.yaml

```diff
@@ -466,10 +474,17 @@ elasticsearch:
     - name: xpack.security.enabled
       value: "false"
 
+  # Disable service links that cause a slow startup.
+  enableServiceLinks: false
+
+  rbac:
+    create: true
+    automountToken: false
+
   # This value should be slightly less than 50% of the requested memory.
   esJavaOpts: -Xmx220m -Xms220m
   xpack:
-    enabled: false
+    elasticenabled: false
```

Comment:
This does not seem right. In fact, I can't find any xpack-propery defined in the es-chart values.yaml file.

Reply:
definitely a typo, thanks for catching that!